packages/elements/src/react/common/form/index.tsx

```diff
@@ -717,7 +717,9 @@ const FieldError = React.forwardRef<FormFieldErrorElement, FormFieldErrorProps>(
       return null;
     }
 
-    const child = typeof children === 'function' ? children(error) : children;
+    const isRenderFunction = typeof children === 'function';
```

Comment:
One possible issue with the approach is if for some reason the user does not return a single element, they will not get the id forwarded.

```tsx
<FieldError>{({ msg }) => <><p>{msg}</p></>}</FieldError>

// <p>Error message</p>
```

Reply:
not applicable with this change https://github.com/clerk/javascript/pull/3396/commits/db24809aac8c807eb959b0674c7fcf0c7159f1de